data/events.txt

```diff
@@ -1078,65 +1078,105 @@ event "fw safe passage ends"
 
 event "navy using mark ii ships"
 	fleet "Small Republic"
-		variant 4
+		remove variant 6
```

Comment:
Yes, I know. I had added them for clarity - without looking back to the original fleet definition, one knows the original variant's weight.